### PR TITLE
PW-1022 Fix for Ratepay

### PR DESCRIPTION
**Pull request by cyattilakiss**
**Description**
add ratepay logo
fix format invoice line item amounts

### CODE REVIEW

File: Gateway/Request/CheckoutDataBuilder.php [hunk 1]
```diff
@@ -211,9 +211,10 @@ protected function getOpenInvoiceData($order)
             $discountAmount += $item->getDiscountAmount();
 
             $priceExcludingTax = $item->getPriceInclTax() - $item->getTaxAmount();
+            $formattedPriceExcludingTax = $this->adyenHelper->formatAmount($priceExcludingTax, $currency);
 
             $formFields['lineItems'][] = [
-                'amountExcludingTax' => $priceExcludingTax,
+                'amountExcludingTax' => $formattedPriceExcludingTax,
                 'taxAmount' => $item->getTaxAmount(),
```
Comment (rikterbeek) [inline Gateway/Request/CheckoutDataBuilder.php]: just wandering if this does not need to be formatted as well ?

File: Gateway/Request/CheckoutDataBuilder.php [hunk 2]
```diff
@@ -246,11 +247,12 @@ protected function getOpenInvoiceData($order)
...
 
             $priceExcludingTax = $this->quote->getShippingAddress()->getShippingAmount() - $this->quote->getShippingAddress()->getShippingTaxAmount();
+            $formattedPriceExcludingTax = $this->adyenHelper->formatAmount($priceExcludingTax, $currency);
 
             $taxClassId = $this->taxConfig->getShippingTaxClass($this->storeManager->getStore()->getId());
 
             $formFields['lineItems'][] = [
-                'amountExcludingTax' => $priceExcludingTax,
+                'amountExcludingTax' => $formattedPriceExcludingTax,
                 'taxAmount' => $this->quote->getShippingAddress()->getShippingTaxAmount(),
```
Comment (rikterbeek) [inline Gateway/Request/CheckoutDataBuilder.php]: same here, should we format ?

File: Gateway/Request/CheckoutDataBuilder.php (hunk 1 above)
Comment (cyattilakiss) [reply Gateway/Request/CheckoutDataBuilder.php]: good one - I will add it

File: Gateway/Request/CheckoutDataBuilder.php (hunk 2 above)
Comment (cyattilakiss) [reply Gateway/Request/CheckoutDataBuilder.php]: same here :)

**Pull request merged**